suites/nautilus/upgrades/upgrade_4.x-1_to_4.x.yaml

```diff
@@ -0,0 +1,100 @@
+tests:
+- test:
+    name: install ceph pre-requisites
+    module: install_prereq.py
+    abort-on-fail: True
+- test:
+    name: ceph ansible install rhcs 4.x-1 from cdn
+    polarion-id: CEPH-83571467
+    module: test_ansible.py
+    config:
+      use_cdn: True
+      build: '4.x'
+      ansi_config:
+        ceph_origin: repository
+        ceph_repository: rhcs
+        ceph_repository_type: cdn
+        ceph_rhcs_version: 4 
+        ceph_stable_release: nautilus
+        osd_scenario: lvm
+        osd_auto_discovery: False
+        ceph_docker_registry_auth: false
+        dashboard_enabled: false
+        copy_admin_key: True
+        ceph_conf_overrides:
+          global:
+            osd_pool_default_pg_num: 128
+            osd_default_pool_size: 2
+            osd_pool_default_pgp_num: 128
+            mon_max_pg_per_osd: 4096
+            mon_allow_pool_delete: True
+          client:
+            rgw crypt require ssl: false
+            rgw crypt s3 kms encryption keys: testkey-1=YmluCmJvb3N0CmJvb3N0LWJ1aWxkCmNlcGguY29uZgo=
+              testkey-2=aWIKTWFrZWZpbGUKbWFuCm91dApzcmMKVGVzdGluZwo=
+    desc: test cluster 4.x-1 cdn setup using ceph-ansible
```

Comment:
you dont to need to say 4.x-1 here , jus make it 4.x 

the upgrade here is frm 4.x cdn to 4.x latest(internal repos right?), if so rename the suite name , remove x-1 convention, ( you dont want to be confused between 4.0 ,4.1 , we dont test 4.0 anymore)

Reply:
this suite shud be compatible with both rhel7 and rhel 8?

Reply:
@XDsubhash the suite is compatible with both rhel7 and 8.

Reply:
Also as the 4.1z1 builds are not yet out the CDN and latest repo points to the same ceph-ansible version. I think it can be re-tested in z1.

Reply:
these are the latest logs for the upgrade to 4.1z1 http://magna002.ceph.redhat.com/cephci-jenkins/cephci-run-1592551071336/

Reply:
@Ameenasuhani can you add both rhel7,rhel8 cli to qe-ci jenkins doc?

Reply:
@XDsubhash I have added  the cli to QECI doc under upgrade section please have a look.